dist/fyta-plant-card.js

```diff
@@ -405,16 +1475,100 @@ export class FytaPlantCardEditor extends LitElement {
+      show_light: true,
+      light_order: "2",
+      show_moisture: true,
+      moisture_order: "1",
+      show_temperature: true,
+      temperature_order: "3",
+      show_salinity: false,
+      salinity_order: "5",
+      show_nutrition: true,
+      nutrition_order: "4"
+    };
+    
+    // Copy values from provided config
+    if (config) {
+      Object.keys(config).forEach(key => {
+        if (config[key] !== undefined && 
+            key !== 'display_options' &&
+            key !== 'show_ec' &&
+            key !== 'sensor_order' &&
+            !key.includes('_priority') &&
+            !key.includes('_position')) {
+          newConfig[key] = config[key];
+        }
+      });
+      
+      // Handle legacy config with sensor_order
+      if (config.sensor_order && config.sensor_order !== 'default') {
+        // Map from old-style ordering to new order values
+        const orderMap = {
+          'l': 'light',
+          'm': 'moisture',
+          't': 'temperature',
+          's': 'salinity',
+          'n': 'nutrition'
+        };
+        
+        const order = config.sensor_order.split('');
+        order.forEach((letter, index) => {
+          if (orderMap[letter]) {
+            const sensorType = orderMap[letter];
+            newConfig[`${sensorType}_order`] = String(index + 1);
+          }
+        });
+      }
+      
+      // Handle legacy config with priority or position values
+      const sensorTypes = ['light', 'moisture', 'temperature', 'salinity', 'nutrition'];
+      sensorTypes.forEach(type => {
+        const priorityKey = `${type}_priority`;
+        const positionKey = `${type}_position`;
+        const orderKey = `${type}_order`;
+        if (config[priorityKey] !== undefined) {
+          newConfig[orderKey] = String(config[priorityKey]);
+        } else if (config[positionKey] !== undefined) {
+          newConfig[orderKey] = String(config[positionKey]);
+        }
+      });
+    }
+    
+    if (newConfig.device_id !== "" && newConfig.title === "" && this.hass) {
+      try {
+        const device = this.hass.devices[newConfig.device_id];
+        if (device && device.name) {
+          newConfig.title = device.name;
+        }
+      } catch (e) {
+        console.error("Error setting title from device name:", e);
+      }
+    }
 
-    if (config.title === "" && config.device_id != "" && this.hass) {
-      const device = this.hass.devices[config.device_id];
-
-      config.title = device.name;
+    // Ensure at least one option is selected
+    const hasSelection = 
+      newConfig.show_light || 
+      newConfig.show_moisture || 
+      newConfig.show_temperature || 
+      newConfig.show_salinity ||
+      newConfig.show_nutrition;
+    
+    if (!hasSelection) {
+      // Default to showing all if nothing selected
+      newConfig.show_light = true;
+      newConfig.show_moisture = true;
+      newConfig.show_temperature = true;
+      newConfig.show_salinity = true;
```

Comment:
```suggestion
      newConfig.show_salinity = false;
```
Shouldn't we set salinity to false, if we disable it in the default config?

Reply:
True. Changed in the latest commit.